features/creating_order/placing_order_after_going_back_from_order_preview.feature

```diff
@@ -12,15 +12,18 @@ Feature: Going back from the order preview
         And there is a customer account "jon.snow@the-wall.com"
         And I am logged in as an administrator
 
-    Scenario: Having filled data after going back from the order preview
+    Scenario: Placing order after going back from the order preview
         When I create a new order for "jon.snow@the-wall.com"
         And I add "Stark Coat" to this order
         And I specify this order shipping address as "Ankh-Morpork", "Frost Alley", "90210", "United States" for "Jon Snow"
         And I select "Free" shipping method
         And I select "Cash on Delivery" payment method
         And I place this order
         And I go back to the order creation
-        Then the "Stark Coat" product should be added
-        And the address "Jon Snow", "Frost Alley", "90210", "Ankh-Morpork", "United States" should be specified as shipping address
-        And the "Free" shipping method should be selected
-        And the "Cash on Delivery" payment method should be selected
+        And I place and confirm this order
+        Then I should be notified that order has been successfully created
+        And this order shipping address should be "Jon Snow", "Frost Alley", "90210", "Ankh-Morpork", "United States"
+        And this order billing address should be "Ned Stark", "Elm Street", "444", "Rivendell", "United States"
```

Comment:
Should be also `"Jon Snow", "Frost Alley", "90210", "Ankh-Morpork", "United States"` and the test will pass :)